inscriptions/ISic000128.xml

```diff
@@ -192,22 +204,21 @@
         <body>
             <div type="edition" xml:space="preserve" xml:lang="la" resp="#TA">
                 <ab>
-                    <lb n="1"/><expan><abbr>P</abbr><ex>ublius</ex></expan> <g ref="#interpunct">·</g> Antistius <g ref="#interpunct">·</g> <expan><abbr>P</abbr><ex>ubli</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>f</abbr><ex>ilius</ex></expan>
+                    <lb n="1"/><persName type="attested"><expan><abbr>P</abbr><ex>ublius</ex></expan></persName> <g ref="#interpunct">·</g> <persName type="attested">Antistius</persName> <g ref="#interpunct">·</g> <persName type="attested"><expan><abbr>P</abbr><ex>ubli</ex></expan></persName> <g ref="#interpunct">·</g> <expan><abbr>f</abbr><ex>ilius</ex></expan>
                     <lb n="2"/><expan><abbr>Cla</abbr><ex>udia</ex></expan>
-                    <lb n="3"/><expan><abbr>Cavar</abbr><ex>ianus</ex></expan>
+                    <lb n="3"/><persName type="attested"><expan><abbr>Cavar</abbr><ex>ianus</ex></expan></persName>
                 </ab>
             </div>
             <div type="apparatus">
                 <listApp>
-                    <app><note>Text of ILTermini</note></app>
+                    <app><note>Text from autopsy</note></app>
```

Comment:
It would be good to record at this point any divergences from the ILTermini text, in the form, e.g. `<app loc="1"><note>Bivona: wibble</note></app>`